api/base/serializers.py

```diff
@@ -29,7 +29,7 @@ def _url_val(val, obj, serializer, **kwargs):
         return val
 
 
-class LinksField(ser.Field):
+class LinksFieldWIthSelfLink(ser.Field):
     """Links field that resolves to a links object. Used in conjunction with `Link`.
```

Comment:
This docstring no longer applies to this class. It should be moved, and a new docstring should be added for this new class.